Select the appropriate genome and whether you are using ERCC spike-in for data normalization. You can enter multiple groups for comparison by checking the ‘Add Groups’ button. Paste the path to samples from each group into an individual box, separated by a comma. Provide a name for each group in the lower text box.<br>
The Cufflinks pipeline is utilized to quantitate expression in FPKM, which can be found in the ‘cuffnorm’ output folder in the file “genes.fpkm_table”. Additional analyses are automatically performed to show basic QC and file comparisons, generate output files for use in GSEA and perform group comparisons. Be aware, the statistics used for quantitation here do not account for multiple comparisons. For an appropriate statistical test accounting for multiple comparisons, we recommend using edgeR or DESeq2. These programs can be run through a user-friendly interface on your computer, such as <a href="https://www.bioinformatics.babraham.ac.uk/projects/seqmonk/">here</a>, by importing bam files generated after the Align step.
